crates/symbolicator/src/sources.rs

```diff
@@ -458,9 +458,33 @@ impl FileType {
     #[inline]
     pub fn from_object_type(ty: ObjectType) -> &'static [Self] {
         match ty {
-            ObjectType::Macho => &[FileType::MachDebug, FileType::MachCode, FileType::Breakpad],
-            ObjectType::Pe => &[FileType::Pdb, FileType::Pe, FileType::Breakpad],
-            ObjectType::Elf => &[FileType::ElfDebug, FileType::ElfCode, FileType::Breakpad],
+            // There are instances where applications have been cross-compiled for some target
```

Comment:
```suggestion
            // There are instances where applications have been compiled for some target
```
I don't think we can claim this is cross-compiling

Reply:
fixed 👍